### DISCUSSION TITLE
ModuleNotFoundError: No module named 'trainer'

### DISCUSSION THREAD

**Discussion by Huba2004** [coqui-ai/TTS#1742] [General Q&A]

**Comment by Teg3z** [comment]
I had the same error when i installed CoquiTTS for the first time by these instructions:

$ git clone https://github.com/coqui-ai/TTS
$ cd TTS
$ pip install -e .

except I forgot to run the "pip install -e ." command... Then it couldnt find the trainer module, so thats how I fixed this error in my stupid case, maybe it helps someone.
